Presentation.Web/app/shared/main-contract/main-contract-section.view.html

```diff
@@ -0,0 +1,25 @@
+﻿<div id="selectMainContractSection" class="object-section-container" ng-if="ctrl.isVisible">
```

Comment:
kill the ng-if and let the parent component decide since that is the one who uses the component

Reply:
also it saves all of the additional evaluation